manifest.json

```diff
@@ -157,6 +157,21 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/centos/7/source-build",
+              "os": "linux",
+              "osVersion": "centos7",
+              "tags": {
+                "centos-7-source-build-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "centos-7": {
```

Comment:
A local tag is only needed if there's another Dockerfile that needs to reference this image as its base image and needs a stable tag in order to do so.  Since there isn't such a Dockerfile that has a dependency on it, this tag can be removed.